installation/kubernetes/helm/openclarity/templates/cr-discovery-server/daemonset.yaml

```diff
@@ -40,12 +40,32 @@ spec:
             - mountPath: /var/lib/docker
               name: docker
               readOnly: true
+            - mountPath: /var/run/crio
+              name: crio
+              readOnly: true
+            - mountPath: /var/lib/containers
+              name: varlibcontainers
+            - mountPath: /var/run/containers
+              name: varruncontainers
+            - mountPath: /tmp
+              name: tmpfs
```

Comment:
do we need to mount tmp here? since tmp store is only used by discoverer to save the image into, while the transfer of the image across service is actually performed via Echo server which means that we don't need tmp mount since the discoverer can write to its own tmp